azure/sdk-identity-resources-storage/Program.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Threading.Tasks;
+
+using Azure;
+using Azure.Identity;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.Storage;
+using Azure.ResourceManager.Storage.Models;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+
+using AccessTier = Azure.ResourceManager.Storage.Models.AccessTier;
+using Sku = Azure.ResourceManager.Storage.Models.Sku;
+
+namespace AzureIdentityStorageExample
+{
+    class Program
+    {
+        static async Task Main(string[] args)
```

Comment:
```suggestion
        static async Task Main()
```

If `args` are not going to be used, these can be omitted from `Main`.